### PR TITLE
app-forensics/chkrootkit: add patch for bug 509000, remove obsolete ebuilds

### PR DESCRIPTION
**Pull request by JordyZomer**
Hey,

I added  a patch for bug 509000. 

Bug: https://bugs.gentoo.org/509000
Closes: https://bugs.gentoo.org/509000

Since this repo was polluted with obsolete ebuild data I cleaned that up.

The package has no maintainer so I added myself as a proxied maintainer.

Kind Regards,

Jordy Zomer

### CODE REVIEW

File: app-forensics/chkrootkit/metadata.xml [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+    <email>admin@jordyzomer.nl</email>
+    <name>Jordy Zomer</name>
+  </maintainer>
+  <use>
 		<flag name="cron">Install cron script for weekly rootkit scans</flag>
 	</use>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
```
Comment (titanofold) [inline app-forensics/chkrootkit/metadata.xml]: All maintainers should be listed together.

File: app-forensics/chkrootkit/metadata.xml [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
-	<!-- maintainer-needed -->
-	<use>
+  <maintainer type="person">
+    <email>admin@jordyzomer.nl</email>
+    <name>Jordy Zomer</name>
+  </maintainer>
+  <use>
 		<flag name="cron">Install cron script for weekly rootkit scans</flag>
 	</use>
+  <maintainer type="project">
```
Comment (titanofold) [inline app-forensics/chkrootkit/metadata.xml]: Reindent. Don't mix tabs and spaces.

**Comment by mgorny** [comment]
Ping.

**Comment by flappyports** [comment]
@JordyZomer do you still intend to fix this based on the above comments?

**Pull request closed without merging**